CONTRIBUTING.md

```diff
@@ -108,3 +109,5 @@ Code coverage of the tests is reported through [codecov](https://codecov.io/gith
 Travis also automatically builds and publishes the [ProbNum documentation](https://probabilistic-numerics.github.io/probnum/modules.html).
 
 Changes to Travis can be made through the `.travis.yml` file, as well as through `tox.ini` since Travis relies on `tox` for both testing and building the documentation.
+
+ProbNum also uses [GitHub Actions](https://docs.github.com/en/actions) to verify that all pushes and pull requests are compliant with the [*Black*](https://github.com/psf/black) code style.
```

Comment:
I assume in the future we want a reference to pre-commit hooks here?

Reply:
I don't have a good idea of how the future docs will look, but I could imagine linking Black's [explanation to set up pre-commit](https://github.com/psf/black#version-control-integration), [pre-commit](https://pre-commit.com/) directly, and/or an explanation on how to set up pre-commit (`pip install pre-commit`, then `pre-commit install`).

In the same context I would also link Black's page on [editor integration](https://github.com/psf/black/blob/master/docs/editor_integration.md).